l10n_it_ddt/data/mail_template_data.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+
+        <!--Email template -->
+        <record id="email_template_edi_ddt" model="mail.template">
+            <field name="name">DDT- Send by Email</field>
+            <field name="email_from">
+                ${(object.company_id.email and '%s &lt;%s&gt;' % (object.company_id.name, object.company_id.email) or '')|safe}
+            </field>
+            <field name="subject">${object.company_id.name} (${object.ddt_number})
+            </field>
+            <field name="partner_to">${object.partner_id.id}</field>
+            <field name="email_to">
+                ${(object.partner_id.email and '%s &lt;%s&gt;' % (object.partner_id.name, object.partner_id.email) or '')|safe}
+            </field>
+            <field name="report_template" ref="action_report_ddt"/>
+            <field name="report_name">${(object.ddt_number)}</field>
+            <field name="auto_delete" eval="True"/>
+            <field name="model_id" ref="model_stock_picking_package_preparation"/>
+            <field name="lang">${object.partner_id.lang}</field>
+            <field name="body_html"><![CDATA[
+<p>Dear ${object.partner_id.name},</p>
+<p>Here is your document of
+%if object.date:
+<strong>${object.date},</strong>
```

Comment:
To make it perfect, you could try to manage when this `object.date` (respectively `object.date_done`) is not set, because in that case the template would be weird:
```
Here is your document of with shipping date from YourCompany
```

Reply:
Ok! Will have a look later today or tomorrow

Reply:
Right! Wrong branch! Now you have it!

Reply:
It seems to me the problem described by @SimoRubi https://github.com/OCA/l10n-italy/pull/554/files#r204357167 would still be present

Reply:
@eLBati I've double checked it and seems ok to me, no?

Reply:
@robyf70 I made https://github.com/LevelPrime/l10n-italy/pull/1 to fix what I meant in https://github.com/OCA/l10n-italy/pull/554#discussion_r204357167